numpy/_globals.py

```diff
@@ -89,3 +92,19 @@ def __repr__(self):
 
 
 _NoValue = _NoValueType()
+
+class CopyMode(enum.Enum):
```

Comment:
Considering it's a public class, a docstring should probably be added.

Not entirely sure where would be the best place to expose its documentation though. 
Any has suggestions here?